articles/portal/ptl-ref-portal-api.md

```diff
@@ -31,6 +31,14 @@ The entry point URL for the Portal API is:
 
 For information and examples about how to use the Portal API, see [*How to use the UKCloud Portal API*](ptl-how-use-api.md).
 
+## OpenAPI
+
+An OpenAPI definition of the Portal API is in development.
+It can be viewed [interactively using Swagger UI](https://app.swaggerhub.com/apis-docs/ukcloud/UKCloud_Portal/development)
+and in [its raw form](https://github.com/UKCloud/create-vdc-with-edge-portal-api-demo/blob/master/docs/portal-api/openapi.json).
+
+Once complete, it will replace this document.
```

Comment:
Is this true, can we replace this document? Why don't we jsu say it's in development and available here. Then as some point in the future we can decide if we replace it.

Reply:
> Is this true, can we replace this document?

This document (ptl-ref-portal-api.md) will be less useful and harder to use.

> Why don't we just say it's in development and available here?

I'll remove the line